stream-chat-android-client/src/test/java/io/getstream/chat/android/client/ChatClientTest.kt

```diff
@@ -221,4 +232,22 @@ internal class ChatClientTest {
 
         client.getCurrentUser() shouldBeEqualTo updateUser
     }
+
+    @Test
+    @ExperimentalCoroutinesApi
+    fun `Sync with empty cids`() = runTest {
```

Comment:
Could we add tests for the happy path as well?
When the precondition is sucessful (Has any cid) we need to verify the result from the API is returned

Reply:
Good call. Added.